core/domain/stats_services.py

```diff
@@ -295,6 +295,28 @@ def save_stats_model_transactional(exploration_stats):
         _save_stats_model, exploration_stats)
 
 
+def get_view_multi(exploration_ids, version_numbers):
+    """Retrieves the view counts for the given exploration IDs.
+
+        Args:
+            exploration_ids: list(str). The list of exploration IDs to get view
+                counts for.
+            version_numbers: list(int). List of version numbers of explorations.
```

Comment:
I don't understand why we're passing in version numbers; the function signature should be the same as the existing one.

Reply:
EDIT: see https://github.com/oppia/oppia/pull/4201#discussion_r156903427

Reply:
Redundant now.